package.json

```diff
@@ -42,6 +42,7 @@
     "@types/react-highlight-words": "^0.16.1",
     "@types/react-router-dom": "^5.1.5",
     "@types/react-test-renderer": "^17.0.1",
+    "@types/react-virtualized": "^9.21.11",
```

Comment:
Should this be a dev dependency?

Reply:
i dont think this is needed as we used `react-window`